packages/web-components/src/player-component/player-component.ts

```diff
@@ -501,18 +539,56 @@ export class PlayerComponent extends FASTElement {
         const segments = await this.fetchAvailableSegments(start, end);
         // eslint-disable-next-line no-console
         if (segments?.timeRanges?.length) {
-            this.currentDay++;
             const date = new Date(Date.UTC(this.currentYear, this.currentMonth - 1, this.currentDay));
             this.currentDate = date;
             this.datePickerComponent.inputDate = date.toUTCString();
+            this.datePickerComponent.date = date;
             this.updateVODStream(true);
         }
     }
 
+    private disablePrevDay(): null {
```

Comment:
>disablePrevDay [](http://example.com/codeflow?start=12&length=14)

rename these functions to updatePrevDayDisabled()

Reply:
* The name of the function disablePrevDay is changed to 'updatePrevDayDisabled'.
* The name of the function disableNextDay is changed to 'updateNextDayDisabled'.